java-core/src/main/java/ai/certifai/junit5/ex1/BasicMethodAssertion.java

```diff
@@ -1,12 +1,12 @@
-package solution;
+package ai.certifai.junit5.ex1;
 
 /**
  * Demo of Junit5 Test (Class)
- *
+ * Test file path : test\java\ai\certifai\junit5\ex1
  * @author Boon Khai Yeoh
  */
 
-public class MathUtils {
+public class BasicMethodAssertion {
```

Comment:
For the naming of this class you can change back to MathUtils, then the test class naming for this class you can name it TestMathUtils or MathUtilsTest. do the same for GetSetMethodAssertion, change back to Coffee, and the test class you can name it TestCoffee or CoffeeTest